substrate/frame/recovery/src/benchmarking.rs

```diff
@@ -353,5 +370,75 @@ mod benchmarks {
 		Ok(())
 	}
 
+	#[benchmark]
+	fn poke_deposit(n: Linear<1, { T::MaxFriends::get() }>) -> Result<(), BenchmarkError> {
+		let caller: T::AccountId = whitelisted_caller();
+		let lost_account: T::AccountId = account("lost_account", 0, SEED);
+
+		// Fund caller account
+		T::Currency::make_free_balance_be(&caller, BalanceOf::<T>::max_value());
+
+		// 1. Setup recovery config for caller
+		insert_recovery_config_with_max_friends::<T>(&caller);
+
+		// 2. Setup active recovery for lost account
+		setup_active_recovery_with_max_friends::<T>(&caller, &lost_account);
+
+		// 3. Get initial deposits
+		let initial_config = <Recoverable<T>>::get(&caller).unwrap();
+		let initial_config_deposit = initial_config.deposit;
+		let initial_recovery_deposit = T::RecoveryDeposit::get();
+		assert_eq!(
+			T::Currency::reserved_balance(&caller),
+			initial_config_deposit.saturating_add(initial_recovery_deposit)
+		);
+
+		// 4. Artificially increase deposits
+		let increased_config_deposit = initial_config_deposit.saturating_add(2u32.into());
+		let increased_recovery_deposit = initial_recovery_deposit.saturating_add(2u32.into());
+
+		<Recoverable<T>>::try_mutate(&caller, |maybe_config| -> Result<(), BenchmarkError> {
```

Comment:
It is subjective, but I never connected with the `*_mutate` methods and find `let mut foo = Something::get() --> Something::put(foo)` more readable. Just a note.